src/js/controllers/ctrl-my-widgets-collaboration.js

```diff
@@ -252,9 +252,10 @@ app.controller('MyWidgetsCollaborationController', function (
 	}
 
 	const _cancelDemote = (user) => {
+		const accessLevel = String($scope.selected.accessLevel)
 		user.warning = false
 		user.remove = false
-		return (user.access = ACCESS.FULL)
+		return (user.access = accessLevel)
```

Comment:
Not sure about this, but I suspect it's a left over from removing coffee script

```suggestion
		(user.access = accessLevel)
```